nvm.sh

```diff
@@ -2392,50 +2392,50 @@ nvm() {
-      nvm_echo '  nvm current                               Display currently activated version of Node'
-      nvm_echo '  nvm ls [<version>]                        List installed versions, matching a given <version> if provided'
-      nvm_echo '    --no-colors                             Suppress colored output'
-      nvm_echo '    --no-alias                              Suppress `nvm alias` output'
-      nvm_echo '  nvm ls-remote [<version>]                 List remote versions available for install, matching a given <version> if provided'
-      nvm_echo '    --lts                                   When listing, only show LTS (long-term support) versions'
-      nvm_echo '    --lts=<LTS name>                        When listing, only show versions for a specific LTS line'
-      nvm_echo '    --no-colors                             Suppress colored output'
-      nvm_echo '  nvm version <version>                     Resolve the given description to a single local version'
-      nvm_echo '  nvm version-remote <version>              Resolve the given description to a single remote version'
-      nvm_echo '    --lts                                   When listing, only select from LTS (long-term support) versions'
-      nvm_echo '    --lts=<LTS name>                        When listing, only select from versions for a specific LTS line'
-      nvm_echo '  nvm deactivate                            Undo effects of `nvm` on current shell'
-      nvm_echo '  nvm alias [<pattern>]                     Show all aliases beginning with <pattern>'
-      nvm_echo '    --no-colors                             Suppress colored output'
-      nvm_echo '  nvm alias <name> <version>                Set an alias named <name> pointing to <version>'
-      nvm_echo '  nvm unalias <name>                        Deletes the alias named <name>'
-      nvm_echo '  nvm install-latest-npm                    Attempt to upgrade to the latest working `npm` on the current node version'
-      nvm_echo '  nvm reinstall-packages <version>          Reinstall global `npm` packages contained in <version> to current version'
-      nvm_echo '  nvm unload                                Unload `nvm` from shell'
-      nvm_echo '  nvm which [current | <version>]           Display path to installed node version. Uses .nvmrc if available'
-      nvm_echo '  nvm cache dir                             Display path to the cache directory for nvm'
-      nvm_echo '  nvm cache clear                           Empty cache directory for nvm'
+      nvm_echo '  nvm --help                                  Show this message'
+      nvm_echo '  nvm --version                               Print out the installed version of nvm'
+      nvm_echo '  nvm install [-s] [<version>]                Download and install a <version>, [-s] from source. Uses .nvmrc as <version> provider if available'
```

Comment:
i don't think we need all this extra "as version provider" text; if you don't know what .nvmrc is, it's explained elsewhere, and if you do, you know it's providing a version.

Reply:
I have put the 'as <version> provider' to better explain the source of the optional <version> parameter. In any case the important part of this PR are the brackets around the <version> and not this comment. Brackets should be there to keep consistency with other commands of nvm because they denote optionality.

If you agree with the brackets addition and not with the 'as <version> provider' part let me make another PR

Reply:
I agree with the brackets for sure :-) i'm just not as convinced on the "as provider" part.

Please don't make another PR; let's just update this one.

Reply:
Cool! I have updated it.

Regards